freedom-e-sdk/bsp/env/start.S

```diff
@@ -70,10 +70,12 @@ _start:
 	/* Call global constructors */
 	//la a0, __libc_fini_array
 1:
+/*
```

Comment:
So this is removing the libc_init_array, which I presume llvm doesn't have, but what do the other changes have to do with LLVM?

Reply:
lld does not have the I (initialized) attribute, so I had to remove it from the linker script.